docs/intro/localdev.rst

```diff
@@ -16,7 +16,7 @@ It's easy to get started with a local Gel instance. Navigate to the root of your
 
 .. code-block:: bash
 
-  $ gel init
+  $ gel project init
```

Comment:
Can we do this?

```suggestion
  $ gel init
```

This command (an alias to `edb:cmd:'gel project init`) takes care...

^^ might be even better, because you'll have a link to the actual command docs.